compiler/test/dotty/tools/dotc/parsing/ModifiersParsingTest.scala

```diff
@@ -137,7 +137,7 @@ class ModifiersParsingTest {
 
     source = parse("def f(implicit a: Int, b: Int) = ???")
     assert(source.defParam(0).modifiers == List(Mod.Implicit()))
-    assert(source.defParam(1).modifiers == List(Mod.Implicit()))
+    assert(source.defParam(1).modifiers == List()) // FIXME? or is this intended?
```

Comment:
@odersky was this change intended?

Reply:
I mean the change in behavior.

Reply:
I don't think this was intended.

Reply:
It looks like it was intended.  There is a [comment]( https://github.com/lampepfl/dotty/pull/5644/files#diff-b171853db82e3d147044c8af7d368593R2016) that was added saying `keep only flags, so that parameter positions don't overlap`. Therefore the test shoudl be upadted.